### DISCUSSION TITLE
why tmpdir.<RANDOM_ID> gets created when using blobfuse2

### DISCUSSION THREAD

**Discussion by rajsshah86** [Azure/azure-storage-fuse#1394] [Q&A]
Hi
I have a blobfuse2 setup which is uploading the content to premium BLOB storage ( with HNS enabled). In the blobfuse config I have used the type as adls. After upload I can see that there are files with name tmpdir.<RANDOM_ID> getting created . Are these marker files ? Is there a way to disable them without effecting the functionality ?
<img width="317" alt="image" src="https://github.com/Azure/azure-storage-fuse/assets/38250316/a057c774-1e65-4b26-a739-8c5f34cbc831">

**Comment by bhansalivikas** [comment]
These are no marker files and Blobfuse2 is not generating any such file/directory on its own. Shall be something coming from your application itself which is trying to copy data.

**Comment by rajsshah86** [comment]
@vibhansa-msft : Agree that it is created by reposync command that I am executing . However Once that command is finished , I cannot delete this directory no matter what I do. The directory is empty but I get error

`rm: cannot remove '/repos/8.8rhui-microsoft-azure-rhel8/tmpdir.xFry7W': Directory not empty`

Doing a ls -ltra to the directory just before rm -rf command also shows no files/hidden files inside it .

Strange thing is that next time when running workflow this directory is deleted , however because this is new run , the new tmpdir throws now the same error .
